types/suggestions.d.ts

```diff
@@ -3,4 +3,26 @@ export interface Suggestion {
   toString(): string
 }
 
-export function getSuggestedQuery(element: HTMLElement): Suggestion | undefined
+export type Variant =
+  | 'get'
+  | 'getAll'
+  | 'query'
+  | 'queryAll'
+  | 'find'
+  | 'findAll'
+
+export type Method =
+  | 'Role'
+  | 'LabelText'
+  | 'PlaceholderText'
+  | 'Text'
+  | 'DisplayValue'
+  | 'AltText'
+  | 'Title'
+  | 'TestId'
+
+export function getSuggestedQuery(
+  element: HTMLElement,
+  variant: Variant,
```

Comment:
variant defaults to `get`

```suggestion
  variant?: Variant,
```

Reply:
yes I didn't see `function makeSuggestion(queryName, content, {variant = 'get', name}) {`

maybe is better to move the default value of variant to exported `getSuggestedQuery` ?

Reply:
I think it should be there as well. Both methods can have the default initializer.